pyproject.toml

```diff
@@ -16,7 +16,7 @@ classifiers = [
   "License :: OSI Approved :: MIT License",
   "Operating System :: OS Independent",
 ]
-requires-python = ">=3.8"
+requires-python = ">=3.11,<=3.12"
```

Comment:
looks like this is also allowing 3.12, should just be 3.11

Reply:
ah sorry about that